aws_quickstart/main_extended.yaml

```diff
@@ -222,8 +222,8 @@ Resources:
     Properties:
       TemplateURL: !If
         - IsCrossAccountScanning
-        - 'https://<BUCKET_PLACEHOLDER>.s3.amazonaws.com/aws/<VERSION_PLACEHOLDER>/datadog_agentless_delegate_role.yaml'
-        - 'https://<BUCKET_PLACEHOLDER>.s3.amazonaws.com/aws/<VERSION_PLACEHOLDER>/datadog_agentless_scanning.yaml'
+        - 'https://datadog-cloudformation-template.s3.amazonaws.com/aws_agentless/v5.0.0/datadog_agentless_delegate_role.yaml'
+        - 'https://datadog-cloudformation-template.s3.amazonaws.com/aws_agentless/v5.0.0/datadog_agentless_scanning.yaml'
```

Comment:
💭 why pin this instead of getting the latest version from `aws_agentless/version.txt`?